tests/unit/CoreQuoteTest.php

```diff
@@ -0,0 +1,375 @@
+				innerBlocks {
+					... on CoreParagraph{
+						attributes {
+							content
+						}
+					}
+				}
+				attributes {
+					anchor
+					backgroundColor
+					citation
+					className
+					cssClassName
+					fontFamily
+					fontSize
+					gradient
+					layout
+					lock
+					# metadata
+					style
+					textAlign
+					textColor
+					value
+				}
+			}
+
+			query Post( $id: ID! ) {
+				post(id: $id, idType: DATABASE_ID) {
+					databaseId
+					editorBlocks {
+						apiVersion
+						blockEditorCategoryName
+						clientId
+						cssClassNames
+						innerBlocks {
+							name
+						}
+						isDynamic
+						name
+						parentClientId
+						renderedHtml
+						...CoreQuoteBlockFragment
+					}
+				}
+			}
+		';
+		$variables = [
+			'id' => $this->post_id,
+		];
+
+		$actual = graphql( compact( 'query', 'variables' ) );
+		error_log( print_r( $actual, true ) );
+
+		$this->assertArrayNotHasKey( 'errors', $actual, 'There should not be any errors' );
+		$this->assertArrayHasKey( 'data', $actual, 'The data key should be present' );
+		$this->assertArrayHasKey( 'post', $actual['data'], 'The post key should be present' );
+
+		$this->assertEquals( $this->post_id, $actual['data']['post']['databaseId'], 'The post ID should match' );
+
+		// Verify the block data.
+		$block = $actual['data']['post']['editorBlocks'][0];
+
+		$this->assertNotEmpty( $block['apiVersion'], 'The apiVersion should be present' );
+		$this->assertEquals( 'text', $block['blockEditorCategoryName'], 'The blockEditorCategoryName should be text' );
+		$this->assertNotEmpty( $block['clientId'], 'The clientId should be present' );
+
+		$this->assertNotEmpty( $block['innerBlocks'], 'There should be no inner blocks' );
+		$this->assertEquals( 1, count( $block['innerBlocks'] ), 'There should be only one inner block' );
+		$this->assertEquals( 'core/paragraph', $block['innerBlocks'][0]['name'], 'The inner block name should be core/paragraph' );
+
+		$this->assertEquals( 'core/quote', $block['name'], 'The block name should be core/quote' );
+		$this->assertEmpty( $block['parentClientId'], 'There should be no parentClientId' );
+		$this->assertNotEmpty( $block['renderedHtml'], 'The renderedHtml should be present' );
+
+		unset( $block['attributes']['citation'] ); // Tested above.
+		unset( $block['attributes']['className'] ); // Tested above.
+		unset( $block['attributes']['cssClassName'] ); // Tested above.
+
+		unset( $block['attributes']['anchor'] ); // Tested above.
+		unset( $block['attributes']['backgroundColor'] ); // Tested above.
+		unset( $block['attributes']['fontFamily'] ); // Tested above.
+		unset( $block['attributes']['fontSize'] ); // Tested above.
+		unset( $block['attributes']['gradient'] ); // Tested above.
+		unset( $block['attributes']['lock'] ); // Tested above.
+		unset( $block['attributes']['style'] ); // Tested above.
+		unset( $block['attributes']['textColor'] ); // Tested above.
+
+
+		// Verify the attributes.
```

Comment:
Instead of retesting/unsetting props, just query for and test what's needed.

(Remove the unused fields from the custom query/frag)


```suggestion
		$block = $actual['data']['post']['editorBlocks'][0];


		$this->assertEquals( 'core/quote', $block['name'], 'The block name should be core/quote' );


		// Verify the attributes.
```